src/libraries/System.Text.RegularExpressions/gen/UpgradeToRegexGeneratorAnalyzer.cs

```diff
@@ -107,26 +107,16 @@ private static void AnalyzeInvocation(OperationAnalysisContext context, INamedTy
             // code fixer can later use that property bag to generate the code fix and emit the RegexGenerator attribute.
             if (staticMethodsToDetect.Contains(method))
             {
-                string? patternArgumentIndex = null;
-                string? optionsArgumentIndex = null;
-
                 // Validate that arguments pattern and options are constant and timeout was not passed in.
-                if (!TryValidateParametersAndExtractArgumentIndices(invocationOperation.Arguments, ref patternArgumentIndex, ref optionsArgumentIndex))
+                if (!TryValidateParametersAndExtractArgumentIndices(invocationOperation.Arguments))
```

Comment:
Small NIT: The name of the method is technically not accurate any longer, now we are only doing validation of the Parameters so we could rename to `ValidateParameters`